examples/22_define-router/src/entries.tsx

```diff
@@ -207,10 +208,44 @@ export default defineRouter({
       ],
       isStatic: true,
     },
+    {
+      path: [
+        { type: 'literal', name: 'api' },
+        { type: 'literal', name: 'empty' },
+      ],
+    },
   ],
-  handleApi: async () => {
-    return {
-      status: 200,
-    };
+  handleApi: async (path) => {
+    if (path === '/api/hi.txt') {
+      const hiTxt = await readFile('./public/hi.txt');
```

Comment:
Reading file from this place is more confusing, as we can't see if `waku build` emits public/hi.txt.

Either, just embed string like below, or place the file somewhere else.

Reply:
I'll place the file elsewhere so we can have both string example and file examples

Where should it go? src/files/hi.txt?

Reply:
actually, this file should be in `dist`. so, `./private/hi.txt` would be the only solution.

Reply:
So it's good as is then?